code/game/objects/structures/lavaland/ore_vent.dm

```diff
@@ -270,6 +270,7 @@
 		balloon_alert_to_viewers("vent tapped!")
 		icon_state = icon_state_tapped
 		update_appearance(UPDATE_ICON_STATE)
+		qdel(GetComponent(/datum/component/gps))
```

Comment:
If you want to go for bonus points, we may want to make a signal that the GPS component can handle it's own deletion, but for now I think this is acceptable 👍